styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -86,9 +85,15 @@ public String createExecution(RunState runState, final String execName, final Fl
           execMode);
       return runnerId;
     } catch (StatusRuntimeException e) {
-      if (e.getStatus().getCode() == Status.Code.NOT_FOUND) {
-        throw new LaunchPlanNotFound(flyteExecConf, e);
-      } // TODO: Consider other status codes that could be interesting
+      switch (e.getStatus().getCode()) {
+        case ALREADY_EXISTS:
+          // TODO: 🤔 How do we make sure that we tell apart between low probabilities of collisions
+          //  on hashing of styx-run-id --> flyte exec and repetitions of same styx-id execution.
+          //  For now we always assume the later.
```

Comment:
How is this expected to be solved? I believe that we probably just let styx retry and the likelihood of the problem happening again is negligible.
Also what do you mean of `same styx-id execution` ?